### DISCUSSION TITLE
About: the calculation of the length of grain boundaries which misorientaion angle between 15 and 45 degree

### DISCUSSION THREAD

**Discussion by Ice-cream19** [mtex-toolbox/mtex#1797] [Ask Anything] [answered]
how to calculate grain boundary length? who can provide me related codes?

**Comment by lfgmorales** [comment, marked as answer]
https://mtex-toolbox.github.io/BoundaryProperties.html

**Reply by Ice-cream19** [reply]
Thank you so much. I have resolved this problem.(*^_^*)
